runtime/Cargo.toml

```diff
@@ -4,7 +4,7 @@ edition = '2018'
 name = 'joystream-node-runtime'
 # Follow convention: https://github.com/Joystream/substrate-runtime-joystream/issues/1
 # {Authoring}.{Spec}.{Impl} of the RuntimeVersion
-version = '7.5.1'
```

Comment:
Why did you update the runtime version?

Reply:
I was testing correct behavior when runtime code changes, with respect to rebuilding joystream/node image, and caching new images.